core/modules/parser/QSMySqlListener.cc

```diff
@@ -2165,6 +2168,10 @@ class UidAdapter :
                    "If value is not set by callback then one of the terminal nodes should be populated.",
                     _ctx);
             _val = _ctx->getText();
+            assertExecutionCondition(__FUNCTION__, (_val.find('`') == 0) && (_val.rfind('`') == _val.size()-1),
+                    "REVERSE QUOTE values should begin and end with a backtick(`).", _ctx);
+            _val.erase(_val.begin());
+            _val.erase(--(_val.end()));
```

Comment:
I'm not familiar with the code. Is there a check to find back ticks in the middle of _val, or does that happen elsewhere in parsing? Is there a test for that?

Reply:
a backtick in the middle wouldn't appear here.